euicc/logger.c

```diff
@@ -0,0 +1,74 @@
+#include "logger.h"
+
+#include <string.h>
+#include <unistd.h>
+
+#define BASH_RED 31
+#define BASH_GREEN 32
+#define BASH_CLEAR 0
+
+static void bash_colorize(FILE *fp, const int color) {
+#if _WIN32
+    // disable bash colorize on non-unix platform
+    (void)fp;
+    (void)color;
```

Comment:
Use `_isatty()` to detect ConPTY on Windows.

Reply:
Windows needs use `SetConsoleMode` api set `ENABLE_VIRTUAL_TERMINAL_INPUT`, but in CI not found `consoleapi.h` file 

https://learn.microsoft.com/en-us/windows/console/setconsolemode

Reply:
https://stackoverflow.com/a/52608157